scripts/aws/cfn/microservice.yml

```diff
@@ -338,19 +344,9 @@ Resources:
               - s3:PutObjectRetention
               - s3:PutObjectTagging
               - s3:PutObjectVersionTagging
-            Resource:
-              - !Sub '${LegalFactsBucketArn}/*'
           - Effect: Allow
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

IAM Policy statement missing Resource or NotResource